miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/EditLibraryController.java

```diff
@@ -687,7 +687,7 @@ protected LibraryDto createDtoFromParent(Sample item) {
       templatesByProjectId = results.stream()
           .map(sam -> sam.getProject().getId())
           .distinct()
-          .map(projectId -> templateService.listLibraryTemplatesForProject(projectId))
+          .map(projectId -> libraryTemplateService.listLibraryTemplatesForProject(projectId))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)